subsys/llext/llext_link.c

```diff
@@ -168,7 +168,7 @@ static void llext_link_plt(struct llext_loader *ldr, struct llext *ext, elf_shdr
 			ret = llext_read(ldr, &rela, sizeof(rela));
 		}
 
-		if (ret < 0) {
+		if (ret != 0) {
```

Comment:
This just looks odd I have to say, but I get the reasoning

Reply:
@pillo79 is that allowed by the API? In fact shouldn't we move `llext_seek()` and friends from under `@cond ignore` and properly document them, thus specifying what they should return? If a function can only return 0 or a negative error, I think it's common to check `< 0` and `== 0` only, leaving impossible `> 0` uncovered